app/src/main/java/com/activities/AddFaceActivity.java

```diff
@@ -283,7 +295,7 @@ private Bitmap preProcessFace(Bitmap image) {
     /**
      * Convert detected face to bitmap and display it on face image View.
      *
-     * @param face detected face.
+     * @param face - Detected face.
```

Comment:
Dashes are not good idea in Javadocs. In autogenerated documentation there is dash added on default and adding dash in here will result in something like this: 
![image](https://user-images.githubusercontent.com/52603374/139830071-8dd91ebd-af53-43a4-8815-9c266d52ab86.png)